app-testing/Makefile

```
@@ -32,24 +32,45 @@ format:
 
 .PHONY: test-ci
 test-ci:
-	pipenv run python -m pytest -m "emulated_alpha"
+	python -m pipenv run python -m pytest -m "emulated_alpha"
 
 .PHONY: test-protocol-analysis
 test-protocol-analysis:
 	pipenv run python -m pytest -v tests/protocol_analyze_test.py
 
 .PHONY: setup
 setup:
```

I don't think it actually matters in this case, but as a general best practice, if `install-pipenv` needs to happen before `setup`, the "proper way" to express that in Make is as a dependency, instead of putting `$(MAKE) install-pipenv` in the recipe.

```suggestion
setup: install-pipenv
```

https://accu.org/journals/overload/14/71/miller_2004/